system/Email/Email.php

```diff
@@ -1877,7 +1890,8 @@ protected function SMTPConnect()
 		{
 			$this->sendCommand('hello');
 			$this->sendCommand('starttls');
-			$crypto = stream_socket_enable_crypto($this->SMTPConnect, true, STREAM_CRYPTO_METHOD_TLS_CLIENT);
+			$crypto = stream_socket_enable_crypto($this->SMTPConnect, true, STREAM_CRYPTO_METHOD_TLSv1_0_CLIENT |
+				STREAM_CRYPTO_METHOD_TLSv1_1_CLIENT | STREAM_CRYPTO_METHOD_TLSv1_2_CLIENT);
```

Comment:
Would it make any difference if we used `STREAM_CRYPTO_METHOD_ANY_CLIENT` here instead?

Reply:
That's an excellent question! `STREAM_CRYPTO_METHOD_ANY_CLIENT` would also enable the deprecated SSL 2.0 and 3.0 ciphers (https://tools.ietf.org/html/rfc7568) and I am not sure what the framework's policy on that is.
Deprecation of TLS 1.0 and 1.1 seems to be underway with the major browsers phasing out support during 2020 (https://www.entrustdatacard.com/blog/2018/november/deprecating-tls)
TLS 1.3 seems to be unsupported by PHP even though openssl supports it, but I haven't been able to understand the implicit support (i.e. is openssl trying to negotiate the highest possible version etc?)

Reply:
OK, thanks. We can leave it as it is.